### PR TITLE
Update Maintainers.md

### PR DESCRIPTION
**Pull request by tysonjhayes**
Modifying to help wanted which is whats actually used

### CODE REVIEW

File: Maintainers.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ If you are maintainer, please follow these rules:
   1. In some cases, special labels may be used (e.g. ```new module``` for issue requesting new DSC resource module in DscResources repository)
   1. If issue is duplicate of another issue, ```duplicate``` label is added and issue is closed
   1. If for some reason issue should not be fixed, ```wontfix``` label is added and comment with explaination should be provided.
-1. Maintainer assigns ```up for grabs``` label to let contributors know this issue needs someone who will take care of it
+1. Maintainer assigns ```help wanted``` label to let contributors know this issue needs someone who will take care of it
 1. Once one of the maintainers or contributors volunteers to work on given issue, ```help wanted``` label is replaced with ```in progress```
```
Comment (KarolKaczmarek) [inline Maintainers.md]: Unfortunately we are going away from 'help wanted' in favor of 'up for grabs', as in PSScriptAnalyzer.
Can you revert this change and also update line 42 with 'up for grabs'? 
Thanks

**Comment by tysonjhayes** [comment]
Done.
